<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>statusBancar</class>
 <widget class="QDialog" name="statusBancar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statut bancaire</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background: rgb(168, 0, 0);
}</string>
  </property>
  <widget class="QLabel" name="valoareLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 18px;
}</string>
   </property>
   <property name="text">
    <string>Valeur:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valoareLineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 14px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="datorieLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 18px;
}</string>
   </property>
   <property name="text">
    <string>Dette:</string>
   </property>
  </widget>
  <widget class="QLabel" name="rataLunaraLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 18px;
}</string>
   </property>
   <property name="text">
    <string>Tarif mensuel:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="datorieLineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 14px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rataLunaraLineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 14px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setarePINButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>310</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 18px;
}</string>
   </property>
   <property name="text">
    <string>Changer le code PIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="codBancarLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 18px;
}</string>
   </property>
   <property name="text">
    <string>Code bancaire:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="codPrimaLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="codADouaLineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>250</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color: white;
}</string>
   </property>
   <property name="text">
    <string>  -</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color: white;
}</string>
   </property>
   <property name="text">
    <string>  -</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>486</x>
     <y>257</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color: white;
}</string>
   </property>
   <property name="text">
    <string>  -</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="codATreiaLineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="codAPatraLineEdit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>390</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 18px;
}</string>
   </property>
   <property name="text">
    <string>Voir les transactions</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>301</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="euro_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 15px;
}</string>
   </property>
   <property name="text">
    <string>euro</string>
   </property>
  </widget>
  <widget class="QLabel" name="euro_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>430</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 15px;
}</string>
   </property>
   <property name="text">
    <string>euro</string>
   </property>
  </widget>
  <widget class="QLabel" name="euro_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 15px;
}</string>
   </property>
   <property name="text">
    <string>euro</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
